### ISSUE TITLE
Prime array lengths. primes themselves, and buckets are computed runtime

### ISSUE THREAD

**Issue by khovratovich** [ligeroinc/LigeroRSA#17]
For example here
https://github.com/ligeroinc/LigeroRSA/blob/15a5950db39fbdfe9f95de382c4602dc5a2e5fd7/include/EncryptedClient.hpp#L2811

However, it seems that both array lengths and values can be computed at compile time. Using predefined constants for them would make the code easier to read and analyze. Currently it is non obvious that buckets are the same for all clients.

**Comment by ligeroinc** [comment]
Agreed. At some point we wanted to have the candidate modulus size parameterizable and we were computing this runtime. But later we decided to hardcode them.
